concent_api/delete-unsupported-messages.py

```diff
@@ -0,0 +1,40 @@
+import django
+import os
+import sys
+
+from logging import getLogger
+
+from common.helpers import deserialize_message
+
+from golem_messages.exceptions import MessageError
+
+sys.path.append('concent_api')
+os.environ['DJANGO_SETTINGS_MODULE'] = "concent_api.settings"
+django.setup()
+
+from core.models import StoredMessage  # noqa E402
+from core.models import Subtask  # noqa E402
+
+logger = getLogger(__name__)
+
+
+def main() -> None:
+    for subtask in Subtask.objects.all():
+        try:
+            report_computed_task = deserialize_message(subtask.report_computed_task.data.tobytes())
+            if not isinstance(report_computed_task.size, int):
+                delete_unsupported_messages(subtask)
+        except MessageError as golem_messages_exception:
+            logger.info(f'During message deserialization exception raised: {golem_messages_exception}')
+            delete_unsupported_messages(subtask)
+
+
+def delete_unsupported_messages(subtask: Subtask) -> None:
+    subtask_id = subtask.subtask_id
```

Comment:
Please don't create intermediate variables for short and simple stuff like this. It does not make it easier to read. It only adds an unnecessary level of indirection.

Reply:
Actually that's a really good point. But you should have added a comment explaining your intention because it's not obvious. As you can see, without a comment everyone thought that your intention was just readability.

Anyway, yeah - Django does not clear the ID if you delete the object so it still wasn't not necessary.